index.html

```diff
@@ -44,21 +44,24 @@
       <h1 class='h1'>Twin Cities Aid Map</h1>
     </div>
     <div class="filter-pane white-and-blue">
-      <p class='p txt-small'>See this data in <a class="txt-white" href="https://docs.google.com/spreadsheets/d/e/2PACX-1vSfnmh9CmtVoBMy2k_-NMhhvF7ursBs2L5IPF9VZ1J-l9P71vjDFw7y6Y6E98d5lNeOAEo7leisP0Y8/pubhtml" target="_blank">Google Sheets</a>.</p>
       <div class='legend' id="key"></div>
-      <button id="toggle-button" class="location-list-toggle" onClick="toggleSidePane(this)">Show list of locations</button>
+      <button id="locations-toggle-button" class="location-list-toggle toggle-button" onClick="toggleSidePane(this)">Show list of locations</button>
+      <a href='#help-info'><button id="help-info-toggle-button" class="toggle-button" onClick="toggleHelpInfo(this)">Help/info</button></a>
```

Comment:
Note that `toggleHelpInfo(this)` is not implemented yet, and `<a href='#help-info'>` is here for if we're going to use css `:target`.

Reply:
We just need the tiniest bit of JS here – on click, clear the fragment identifier. We'll need to make sure we also clear the fragment identifier when user clicks on 'show list of locations'.

Reply:
@samanpwbb you're a wizard! 🧙